src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -260,19 +260,19 @@
     <Compile Include="$(BclSourcesRoot)\System\WeakReference.CoreCLR.cs" />
     <Compile Include="$(BclSourcesRoot)\System\WeakReference.T.CoreCLR.cs" />
     <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.GetStdHandle.cs">
```

Comment:
This is not a valid XML.

Also, it would be nice to make it fit the formatting suggested in https://github.com/dotnet/runtime/issues/1266#issuecomment-615338260:
```
<Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.GetStdHandle.cs"
         Link="Common\Interop\Windows\Kernel32\Interop.GetStdHandle.cs" />
<Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.HandleTypes.cs"
         Link="Common\Interop\Windows\Kernel32\Interop.HandleTypes.cs" />
```

Reply:
Yeah sorry, I botched the find/replace. I'll fix this along with the existing Link attributes.